pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -427,6 +434,10 @@ func fetchAndStoreGuestInfo(infoTypes libvirt.DomainGuestInfoTypes, agentPoller
 	if infoTypes&libvirt.DOMAIN_GUEST_INFO_USERS != 0 {
 		agentPoller.agentStore.Store(libvirt.DOMAIN_GUEST_INFO_USERS, convertToUsers(guestInfo))
 	}
+
+	if infoTypes&libvirt.DOMAIN_GUEST_INFO_LOAD != 0 {
+		agentPoller.agentStore.Store(libvirt.DOMAIN_GUEST_INFO_LOAD, convertToLoad(guestInfo))
+	}
```

Comment:
This code once appeared in https://github.com/kubevirt/kubevirt/pull/14983.

In https://github.com/kubevirt/kubevirt/pull/14983#discussion_r2164274536 we were discussing the performance implication of this.